server/routes/notification.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const { validateMongoId, validateNotificationDetails } = require("../validate");
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  markNotificationAsRead,
+  getAllNotifications,
+  getUnreadNotifications,
+} = require("../controllers/notification");
+
+router
+  .route("/create")
+  .post(validateNotificationDetails, createNotification);
+
+router.route("/check/:_id").post(validateMongoId, markNotificationAsRead);
```

Comment:
We will want to `protect` these routes.

I think we can omit the keywords for these routes (e.g. `create/check`). The http verb and the slug conveys a clear intention thanks to the conventions used.